Tests/AppTests/AnalyzerTests.swift

```diff
@@ -211,7 +211,7 @@ class AnalyzerTests: AppTestCase {
 
         // validate score
         XCTAssertEqual(pkg1.score, 35)
-        XCTAssertEqual(pkg2.score, 45)
+        XCTAssertEqual(pkg2.score, 40)
```

Comment:
This show the actual bug in our live (before this change) implementation: pkg2's version has `NULL` in `resolved_dependencies` yet we awarded 5 points because we defaulted to `[]`:

![CleanShot 2023-11-24 at 10 01 53@2x](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/assets/65520/9799f6cf-39aa-4063-b05c-6ba3092c204c)

This is now fixed.

Reply:
I was just about to type, "Why the score change?" 😂